### PR TITLE
Satellite images

### PR DESCRIPTION
**Pull request by ENT8R**
I think this is now ready to merge! Maybe @matkoniecz could look if everything is fine. That would be great 👍

### CODE REVIEW

**Comment by matkoniecz** [comment]
Technical issue: with separate file for  satellite-style.yaml - is there a good reason for a separate branch? 

I would expect global.yaml for config and shared code if any, satellite-style.yaml (satellite image style, probably just labels), base-style.yaml (standard style) on master branch.

**Comment by matkoniecz** [comment]
Also, satellite-style.yaml on satellite-images bramch should not have 

```
import:
  - https://raw.githubusercontent.com/ENT8R/streetcomplete-mapstyle/master/global.yaml
  - https://raw.githubusercontent.com/ENT8R/streetcomplete-mapstyle/master/layers.yaml
```

iff it is supposed on a separate branch. Edpecially as master has currently no global.yaml file.

Also, is there a good reason to split layers.yaml and global.yaml?

**Comment by ENT8R** [comment]
> Also, is there a good reason to split layers.yaml and global.yaml?

I splitted them just because I thought it might be better to read. But it can be one file too.

> Technical issue: with separate file for satellite-style.yaml - is there a good reason for a separate branch?

So you think we should maintain a separate branch for the satellite style and I shouldn't merge this PR? I thought it might be the best to have all the code at one place...

File: global.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    text_font_family: 'Montserrat'
+    text_stroke_color: 'white'
+    text_stroke: { color: global.text_stroke_color, width: 2px }
+    text_fill_color: 'black'
+    building_color: '#caffd2'
+    building_outline: black
+    road_color: '#4da39a'
+    road_outline: black
+    path_color: global.road_color
+    path_outline: global.road_outline
```
Comment (westnordost) [inline global.yaml]: Shouldn't the globals that only apply to the layers style not be moved there?

I understand that global.yaml is designed to be the "shared" code. Then, path_color etc. wouldn't be shared though.

File: global.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        url: https://tile.mapzen.com/mapzen/vector/v1/all/{z}/{x}/{y}.mvt
+        url_params:
+            api_key: global.sdk_mapzen_api_key
+        tile_size: 512
+
+    mapbox-satellite:
+        type: Raster
+        url: https://api.mapbox.com/v4/mapbox.satellite/{z}/{x}/{y}.png
+        url_params:
+            access_token: pk.eyJ1IjoiZW50OHIiLCJhIjoiY2o5aXBvZGI1MGMyZzJxbjJtNGxzZzllYiJ9.7vUh6rE_BUG1RdfEqQdtCg
```
Comment (westnordost) [inline global.yaml]: Same here, I guess

**Comment by matkoniecz** [comment]
> So you think we should maintain a separate branch for the satellite style and I shouldn't merge this PR? I thought it might be the best to have all the code at one place...

Ops, sorry. I misinterpreted PR target/source (I agree that separate branches are not necessary).

**Comment by matkoniecz** [comment]
> I splitted them just because I thought it might be better to read. But it can be one file too.

Now after looking again I see difference between content of each style. Personally I would create single file as size is quite small, but two separate also are OK.

**Comment by ENT8R** [comment]
> Where did you move the labels? I don't see them in this changeset

The address-labels are seperate in each style.
And the place labels are in the imported code.

File: global.yaml (hunk 1 above)
Comment (ENT8R) [reply global.yaml]: This is now deprecated because there is now only one imported file which needs **all** globals

**Comment by matkoniecz** [comment]
I have not tested styles itself, but everything looks OK.

**Pull request merged**